cmd/replicate-add.go

```diff
@@ -62,7 +62,8 @@ var replicateAddFlags = []cli.Flag{
 	},
 	cli.StringFlag{
 		Name:  "replicate",
-		Usage: "comma separated list to enable replication of delete markers, deletion of versioned objects and replica metadata sync(in the case of active-active replication).Valid options are \"delete-marker\", \"delete\" ,\"replica-metadata-sync\", \"existing-objects\" and \"\"",
+		Value: `delete-marker,delete,existing-objects,metadata-sync`,
+		Usage: `comma separated list to enable replication of soft deletes, permanent deletes, existing objects and metadata sync`,
```

Comment:
this usage description change is needed in `mc replicate edit` also

Reply:
will send that in separate PR @poornas